.github/workflows/release.yml

```
@@ -0,0 +1,140 @@
+
+          echo >> release_body.txt
+          echo "**Attachments**" >> release_body.txt
+          echo "File | Contents" >> release_body.txt
+          echo "| --- | --- |" >> release_body.txt
+          echo "| \`doc-v${PROJECT_VERSION}-en.tar.gz\` | English documentation. Redirection to English" >> release_body.txt
+          echo "| \`pgorpy-${PROJECT_VERSION}.tar.gz\` | tar.gz of the release" >> release_body.txt
+          echo "| \`pgorpy-${PROJECT_VERSION}.zip\` | zip of the release" >> release_body.txt
+          cat release_body.txt
+
+      - name: Create Draft Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)